Italian/guardare/vecchie-fotografie/vecchie-fotografie.md

```diff
@@ -0,0 +1,6 @@
+Le fotografie hanno uno scaffale a loro riservato, e la maggiorparte è stata messa con cura all'interno di cornici
+d'argento.
+
+Sembrano fotografie scattate negli anni 50, almeno a giudicare dall'abbigliamento di coloro che vi sono ritratti.
+
+Vi sono uomini e donne di diversa età, e in alcune immagini, inspiegabilmente, vi sono delle bruciature sul viso di alcune persone.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.